src/cdk-experimental/menu/menu-group.ts

```diff
@@ -22,5 +22,15 @@ import {CdkMenuItem} from './menu-item';
 })
 export class CdkMenuGroup {
   /** Emits the element when checkbox or radiobutton state changed  */
-  @Output() change: EventEmitter<CdkMenuItem> = new EventEmitter();
+  @Output() change: EventEmitter<MenuItem> = new EventEmitter();
+
+  /**
+   * Emits events for the clicked MenuItem
+   * @param menuItem The clicked MenuItem to handle
+   */
+  _registerTriggeredItem(menuItem: MenuItem) {
```

Comment:
Would a different verb other than "register" be more descriptive here? Maybe something like `_emitSelectionChangeEvent`